### DISCUSSION TITLE
想下班

### DISCUSSION THREAD

**Discussion by kaijinti** [share121/inter-knot#3294] [其他]
整天上班就想着下班，审图的人都跑去玩黑神话了，图画完了没人审核

**Comment by kaijinti** [comment]
审图完了，给我说他上次的反馈完全错了，他以为是另一个主题

**Comment by yzysrewrwqr** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/9939adf214795d97dad9abd90265aab2.png" alt="帕姆_哭哭" width="75" >还有半小时下班，我也猴急
